v2/tools/generator/internal/codegen/pipeline/apply_cross_resource_references_from_config.go

```diff
@@ -0,0 +1,237 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT license.
+ */
+
+package pipeline
+
+import (
+	"context"
+
+	"github.com/pkg/errors"
+	kerrors "k8s.io/apimachinery/pkg/util/errors"
+	"k8s.io/klog/v2"
+
+	"github.com/Azure/azure-service-operator/v2/tools/generator/internal/astmodel"
+	"github.com/Azure/azure-service-operator/v2/tools/generator/internal/config"
+)
+
+type ARMIDPropertyClassification string
+
+// TODO: I don't like the names I am using here
+const (
+	ARMIDPropertyClassificationUnset       = ARMIDPropertyClassification("unset")
```

Comment:
```suggestion
	ArmIDPropertyClassificationUnset       = ArmIDPropertyClassification("unset")
```

Reply:
Same suggestion for below `set` and `unspecified`

Reply:
Fully capital for acronyms and initialisms is idiomatic Go - so `ARMID` is more correct despite the run-together. (I'm not a big fan, TBH, but that's the convention.)

Reply:
Agree w/ Bevan here.